### PR TITLE
Add TDP 54

### PR DESCRIPTION
**Pull request by mzero2nine**
Forgot this one @dimsome thank you! :)

### CODE REVIEW

File: TDP/tdp-53.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+created: 2022-12-08
+---
+
+## Simple Summary
+
+This proposal requests four (4) months of operational funding for the mStable Ecosystem subDAO. The proposal will:
+
+⦁	Outline a budget proposal for the period of 1st January 2023 to 30th April 2023
+⦁	Define deliverables and methods of transparently reporting activity and performance during the funding period
+⦁	For more background on the mStable Ecosystem subDAO's purpose, mandate, structure, and internal processes; please see the initial funding request here [TDP 46](https://forum.mstable.org/t/tdp-46-mstabledao-restructure-ecosystem-subdao/895)
```
Comment (dimsome) [inline TDP/tdp-53.md:16]: Can we make these bullet points conform to Markdown?

https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md

See bullet list item.

File: TDP/tdp-53.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- Moderator Pay: 25,000
+- Raffles - 6,250
+- Rewards for Completing Surveys - 1,875
+- Tips - 1250
+
+Summary:  83,125 MTA amount with **17,351** requested for this funding round
+
+**Total MTA Requested: 17,351**
+
+# Deliverables
```
Comment (dimsome) [inline TDP/tdp-53.md:89]: How is this an h1 again?

File: TDP/tdp-53.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+Timeline: 1/23 - 4/23. This funding request will carry the Ecosystem subDAO through the first four (4) months of 2023. Monthly report comprising of highlights, lowlights, and ecosystem subDAO performance will be provided monthly to the treasury subDAO.
+
+⦁ Successful utilization of Dework, defined through the regular successful posting, quality assurance, task completion, and payout of various bounties where previously defined deliverables that add value to the mStable ecosystem are met. The ecosystem subDAO will more deeply collaborate with the builder subDAO to increase the value accrual for the mStable community via the use of Dework.
+
+⦁ Hosting of community events: Community Calls, AMA’s, Game Nights, Trivia where community members are rewarded through their participation via one or more of the following: POAP’s, NFT’s, MTA rewards, special discord roles, recognition in official announcements, and so on. Determination of the number and timing of community-based events will be coordinated with the Builder subDAO.
```
Comment (dimsome) [inline TDP/tdp-53.md:95]: List see above.

File: TDP/tdp-53.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Accountability & Transparency
+
+It is suggested that the mStable TreasuryDAO continues to conduct a monthly review to hold the Ecosystem subDAO accountable to the deliverables defined herein.
+
+The Ecosystem subDAO commits to providing the TreasuryDAO access to information on all expenditure for each month prior to the 10th of the following month, including the category of expenditure. This will either be provided via a report/presentation.
+
+# Additional and Future Funding Requests
```
Comment (dimsome) [inline TDP/tdp-53.md:102]: h1 again?

File: TDP/tdp-53.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+It is proposed that $30,741.68 is transferred in full prior to the start of the funding round (January 1st, 2023).
+
+This will allow the subDAO funding to be used immediately for expenses requiring advance payment, and help streamline the accounting process over the course of the funding period.
+The mStableDAO is expected to hold the subDAO accountable for using funds as outlined and for meeting deliverables over the efunding period as defined in this proposal. This process will include a monthly review of the Ecosystem subDAO's activities and performance.
+
+Below is an itemized list of expected costs. Although not anticipated, the buffer amount will provide some flexibility if there is a variance in expenditures beyond the baseline budget outlined herein.
+
+**Itemized Budget**
+
+- Salaries (Total)	- $25,066.68
```
Comment (dimsome) [inline TDP/tdp-53.md:52]: Dunno but let's keep consistency, either use - or *

File: TDP/tdp-53.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+A full itemized list of funding initiatives can be found in the “Amount Requested” portion of this Proposal. Funding received will be utilized to optimally run the subDAO to carry out its initiatives and meet its objectives.
+
+## Motivation
+
+The Ecosystem subDAO Funding Request is created in order to resolve the merger between the Community subDAO and the GrantsDAO ratified in [TDP 46](./tdp-46)
+
+## Specification
+
+**mUSD**
```
Comment (dimsome) [inline TDP/tdp-53.md:40]: we could use h3 here?

File: TDP/tdp-54.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+This proposal requests 4 months of operational funding for the mStableDAO Cat Herder. The proposal will:
+
+- Outline a budget proposal for the period of 1st January 2023 to 30th April 2023
+- Outline Tasks and responsibilities within the jurisdiction of the Cat Herder
+- Outline a streamlined process for rolling over the Cat Herder position in the future
+
+## Abstract
+
+With [TDP 47](https://mips.mstable.org/TDP/tdp-47.html) being resolved by our Meta Governors via [Snapshot](https://vote.mstable.org/#/proposal/0xb88454b3be7719673c65f2e57e64425027a84cf90839c4e2b4b3113ab23c98f3), it was discussed internally how to streamline this process when the current Cat Herder is happy to continue contributing to the DAO, and given the resolution of the above TDP, it was suggested to instead create a sub-process for this scenario where a Snapshot vote is employed to roll over the Cat Herder for another 4 months given a positive resolution.
```
Comment (dimsome) [inline TDP/tdp-54.md:20]: relative path for TDPs

File: TDP/tdp-54.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+With [TDP 47](https://mips.mstable.org/TDP/tdp-47.html) being resolved by our Meta Governors via [Snapshot](https://vote.mstable.org/#/proposal/0xb88454b3be7719673c65f2e57e64425027a84cf90839c4e2b4b3113ab23c98f3), it was discussed internally how to streamline this process when the current Cat Herder is happy to continue contributing to the DAO, and given the resolution of the above TDP, it was suggested to instead create a sub-process for this scenario where a Snapshot vote is employed to roll over the Cat Herder for another 4 months given a positive resolution. 
+
+This will give additional opportunities for the DAO to check the performance of the Cat Herder throughout the year, while also creating a smoother overall process as the rollover falls in line with the Funding Requests of the Builder & Ecosystem subDAO. 
+
+## Motivation
+
+The Cat Herder remains a crucial part of Governance within the mStableDAO ecosystem, and thus it is proposed here to keep funding the Cat Herder entity for an additional 4 months from the TreasuryDAO.
+
+## Specification
```
Comment (dimsome) [inline TDP/tdp-54.md:28]: Empty specs?

File: TDP/tdp-54.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+This Rollover Request will provide an opportunity for feedback on the structure of the role and performance of the Cat Herder, and would allow for alternate proposals to be made should DAO stakeholders feel that changes need to be made to the role.
+
+If the current Cat Herder is being offboarded from the mStableDAO, the regular election process will kick in as agreed in TDP 47 instead of the Rollover Request.
+
+Given how this is slightly different than the agreed upon process in TDP 47, an additional option will be presented in the first Snapshot done in this way to ensure everyone is happy with this approach moving forward.
+
+Similar to the funding requests, the request will stay open for 2 weeks in which Meta Governors can comment and feedback on both the process itself as well as the Cat Herder role and performance.
+
+## Tasks and responsibilities
```
Comment (dimsome) [inline TDP/tdp-54.md:46]: Should this be a h3?

File: TDP/tdp-54.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+16. Management and maintenance of the relationship between the subDAOs & the mStableDAO
+
+17. Assisting in Treasury Management for the mStableDAO
+
+18. Flexible assistance & general support function for the wider mStable ecosystem and their core contributors when required
+
+19. Quality Assurance & Quality Control that subDAOs & Signers keep delivering upon their milestones & targets, and ensuring that the public representatives of the subDAOs communicate their proof of work in time and according to the funding request
+
+## ***Compensation***
```
Comment (dimsome) [inline TDP/tdp-54.md:88]: H3 again? No need to make it bold if it's already a headline

**Review by dimsome — CHANGES_REQUESTED** [review]
some changes please

File: TDP/tdp-54.md (hunk 10 above)
Comment (mzero2nine) [reply TDP/tdp-54.md:88]: Agreed, copy pasted from bad sources 📦

File: TDP/tdp-54.md (hunk 9 above)
Comment (mzero2nine) [reply TDP/tdp-54.md:46]: yessir

File: TDP/tdp-54.md (hunk 8 above)
Comment (mzero2nine) [reply TDP/tdp-54.md:28]: fixed heading below to make sense

File: TDP/tdp-54.md (hunk 7 above)
Comment (mzero2nine) [reply TDP/tdp-54.md:20]: uff again failed there, mb!

File: TDP/tdp-53.md (hunk 6 above)
Comment (mzero2nine) [reply TDP/tdp-53.md]: yessir

File: TDP/tdp-53.md (hunk 4 above)
Comment (mzero2nine) [reply TDP/tdp-53.md]: yas, bad sources copy pasted

File: TDP/tdp-53.md (hunk 3 above)
Comment (mzero2nine) [reply TDP/tdp-53.md]: yes!

File: TDP/tdp-53.md (hunk 2 above)
Comment (mzero2nine) [reply TDP/tdp-53.md]: Dunno xD

File: TDP/tdp-53.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mzero2nine) [reply TDP/tdp-53.md]: Yap!

**Pull request closed without merging**